core/ui/vault/vaultAddressBook/components/emptyAddressesView/EmptyAddressesView.tsx

```
@@ -1,40 +1,39 @@
 import { Button } from '@lib/ui/buttons/Button'
 import { PageHeaderBackButton } from '@lib/ui/page/PageHeaderBackButton'
 import { PageHeaderTitle } from '@lib/ui/page/PageHeaderTitle'
+import { Text } from '@lib/ui/text'
+import { GradientText } from '@lib/ui/text'
```

ESLint

[error] 'GradientText' is defined but never used.

(@typescript-eslint/no-unused-vars)